db_books.rb

```diff
@@ -0,0 +1,26 @@
+require 'json'
+
+module BooksPersistence
+  def store_books(books)
+    data = []
+    file = './database/books.json'
+    return unless File.exist?(file)
```

Comment:
- [ ] Kindly handle the case when one or more of the files are missing (the app should work and show no errors). For now, in your app, when one file or more is missing the data is not preserved, instead, this should create a new file and preserve the date.